File: Oishi/Basics of Programming/Getting Started/Inverse.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main(int argc, char **argv){
    int n;
    cin >> n;

    //write your code here
    int inverted = 0;
    int originalPlace = 1;

    while (n>0) {
        int end = n % 10;
        int invertedDigit = originalPlace;
        int invertedPlace = end;

        inverted += invertedDigit * pow(10, invertedPlace - 1);
        n /= 10;
        originalPlace++;
    }
    cout << inverted << endl;
}

File: V.Vennela/Basics of Programming/Inverse.java
import java.util.*;

public class Inverse
{

public static void main(String[] args) {
  // write your code here  
  
    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();
    int i=1,inverse=0;
    while(n!=0)
    {
        int r = n%10;
        inverse += i*(int)Math.pow(10,r-1);
        i++;
        n=n/10;
    }
    System.out.println(inverse);
 }
}
